shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Address.hs

```diff
@@ -34,7 +34,7 @@ where
 
 import Cardano.Binary (DecoderError (..), FromCBOR (..), ToCBOR (..))
 import qualified Cardano.Crypto.Hash.Class as Hash
-import Cardano.Prelude (NFData, NoUnexpectedThunks, cborError)
+import Cardano.Prelude (NFData, NoUnexpectedThunks, cborError, panic)
```

Comment:
Honestly I'd rather avoid using additional things from `Cardano.Prelude`, since I would like to remove it, ultimately. But I guess this is a minor usage in a module already using it.

Reply:
Should I use `error` instead?